K1ngmar/Transcendence
# Orm/database

### Created `User` and `Match` table with relations their corresponding *API* endpoints:

### How to test:
> make sure to clean the current database because docker sucks `bash run.sh -fclean`
```bash
# create users and matches
$ bash src/scripts/fill_database.sh

# get users (trough localhost:300/api/user/{id}) or use a curl
$ curl -X 'GET' 'http://localhost:3000/user/{id}' \ -H 'accept: */*'
# make sure to change the {id} with the id of the user you want to see

# update user fields ie:
# this will update userid: 2, field: twoFactorEnabled, value: true
$ bash src/scripts/database/update_user.sh 2 "twoFactorEnabled" true

# this will update the username:
$ bash src/scripts/database/update_user.sh 2 "avatar" \"gekke_avatar\"

# you can easily check matches trough Swagger
> http://localhost:3000/api/

```
